.github/workflows/dev-upload-addons-to-jfrog.yml

```diff
@@ -52,61 +42,37 @@ jobs:
       env:
         VERSION: ${{ inputs.new_version }}
 
-    - name: list version info
-      run: 
-        ls aerospike/;
-        ls aerospike/${{ env.VERSION }}/;
-      env:
-        VERSION: ${{ inputs.new_version }} 
-
-    - name: list version info
+    - name: Move bindings to lib/binding
       run: |
         mkdir lib/binding;
         cp -r aerospike/${{ env.VERSION }}/* lib/binding;
       env:
         VERSION: ${{ inputs.new_version }} 
 
-    - name: list version info
-      run: 
-        ls lib/binding;
-      env:
-        VERSION: ${{ inputs.new_version }} 
-
-    - name: list version info
+    - name: Update version in package.json
       run: 
         jq --arg v "${{ env.VERSION }}" '.version = $v' package.json > tmp.json && mv tmp.json package.json;
-        npm install;
       env:
         VERSION: ${{ inputs.new_version }} 
 
-    - name: list version info
-      run: |
-        cp -r aerospike/${{ env.VERSION }}/* lib/binding;
-      env:
-        VERSION: ${{ inputs.new_version }} 
-
-    - name: list version info
-      run: 
-        ls lib/binding;
+    - name: Install dependencies and test install
+      run:
+        npm ci;
```

Comment:
Is it possible to add `--ignore-scripts` here?

Reply:
Added --ignore-scripts and ran npm scripts manually without using npm run.